client/src/components/pages/Home/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
+import './index.css';
+
+const HomePage = () => {
+  return (
+    <div className="home">
+      <div className="home__header">
+        <div className="home__header__img__overlay"></div>
```

Comment:
this class name notation is unconventional, I suggest to use [BEM](http://getbem.com/naming/). The names should be in following form: block__element--modifier (modifier is optional). Block would be `home`, element in this case `header-img-overlay` or just `img-overlay` and no modifier, i.e. `home__img-overlay`

Reply:
Done